docs/pages/workflow/debugging.md

```diff
@@ -55,41 +56,65 @@ This might indicate that there is a performance issue. You likely need to run yo
 
 The Expo community and the React and React Native communities are great resources for help when you get stuck. There's a good chance someone else has run into the exact same error as you, so make sure to read the documentation, search the [forums](https://forums.expo.dev/), [GitHub issues](https://github.com/expo/expo/issues/), and [StackOverflow](https://stackoverflow.com/).
 
-## Useful tools for debugging
+## Debugging Tools
```

Comment:
```suggestion
## Debugging tools
```

Please see my next comment (in the developer menu) on why I am suggesting this to change to use sentence case.